src/main/java/Duke.java

```diff
@@ -1,10 +1,105 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final String COMMAND_EXIT = "bye";
+    public static final String COMMAND_LIST = "list";
+    public static final String COMMAND_MARK_DONE = "done";
+    public static final String COMMAND_NEW_TODO = "todo";
+    public static final String COMMAND_NEW_DEADLINE = "deadline";
+    public static final String COMMAND_NEW_EVENT = "event";
```

Comment:
It is good how you used constants with all all uppercase using underscore to separate words, following the coding standards for constant names.